quickstarts/aws/amazon-security-lake/config.yml

```diff
@@ -0,0 +1,45 @@
+slug: aws-vpc
```

Comment:
Did you mean `aws-security-lake` or so, instead?

Reply:
yes! not sure how I missed that one.